pkgs/top-level/python-packages.nix

```diff
@@ -12500,6 +12500,28 @@ in {
     };
   };
 
+  hkdf = buildPythonPackage rec {
+    name = "hkdf-${version}";
+    version = "0.0.3";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/h/hkdf/${name}.tar.gz";
+      sha256 = "1jhxk5vhxmxxjp3zj526ry521v9inzzl8jqaaf0ma65w6k332ak2";
+    };
+
+    buildInputs = with self; [ nose ];
+
+    checkPhase = ''
+      nosetests
+    '';
```

Comment:
@FRidh any chance you could take a look at this?

In case my previous message wasn't clear: all tests succeed, but `nosetests` for hkdf runs 0 of them.